test_scripts/API/SetAppIcon_backslash/ATF_N_SetAppIcon_SyncFileName_Contains_BackSlashScreened_as_Param.lua

```diff
@@ -0,0 +1,70 @@
+-- Requirement summary:
+--[GENIVI] SDL must respond INVALID_DATA in case mobile app sends "/" (backslash) symbol at "fileName" parameter
+--
+-- Description:
+-- SDL must respond INVALID_DATA in case mobile app sends "/" (backslash) symbol at "fileName" parameter" must be implemented.
```

Comment:
@Aleksandriys, summary and description is about backslash, but slash "/" symbol is used; so requirement summary and description should be updated with correct backslash symbol.